docs/error-api.asciidoc

```diff
@@ -1,11 +1,25 @@
+[[error-api]]
```

Comment:
Note to my self that this will break the Node agent docs because they are linking to the old anchor

Reply:
@watson can you open a PR on the node agent side which is merged at a similar time like this one so we don't break the doc build?

Reply:
Good idea: https://github.com/elastic/apm-agent-nodejs/pull/43

Reply:
@ruflin could you verify that I changed the link correctly? I.e. that the link is going to be like I predicted